### PR TITLE
Feature/sdl 0269 new vehicle data climate data

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
Implements https://github.com/smartdevicelink/rpc_spec/issues/215

This PR is **[ready]** for review.

### Summary
Add new vehicle data ClimateData

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by KhrystynaDubovyk** [comment]
@atiwari9 please review this PR

**Review by atiwari9 — APPROVED** [review]

**Comment by atiwari9** [comment]
Approved contingent on successful test pass of https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2491

**Comment by LitvinenkoIra** [comment]
@JackLivio This PR is ready for Livio review. Thank you!

File: MOBILE_API.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```
Comment (ShobhitAd) [inline MOBILE_API.xml]: The version has already been set to 7.1.0 in the develop branch. Please pull in the latest changes from develop.

**Review by ShobhitAd — CHANGES_REQUESTED** [review]

File: MOBILE_API.xml (hunk 1 above)
Comment (LitvinenkoIra) [reply MOBILE_API.xml]: @ShobhitAd Thank you! Done in https://github.com/smartdevicelink/rpc_spec/pull/299/commits/bb4c7fd6dbc79b9c5999c2cad0cd6aef7c93c6be

**Review by ShobhitAd — APPROVED** [review]

File: MOBILE_API.xml [hunk 2]
```diff
@@ -550,7 +550,12 @@
         <element name="VEHICLEDATA_FUELLEVEL" until="7.0"/>
         <element name="VEHICLEDATA_FUELLEVEL_STATE" until="7.0"/>
         <element name="VEHICLEDATA_FUELCONSUMPTION" />
-        <element name="VEHICLEDATA_EXTERNTEMP" />
+        <element name="VEHICLEDATA_CLIMATEDATA" since="7.1"/>
+        <element name="VEHICLEDATA_EXTERNTEMP" until="7.1">
```
Comment (NicoleYarroch) [inline MOBILE_API.xml]: `VEHICLEDATA_EXTERNTEMP` should be marked as `deprecated="true" since="7.1"`

Comment (jordynmackool) [reply MOBILE_API.xml]: @LitvinenkoIra, can you please make this update?

Comment (LitvinenkoIra) [reply MOBILE_API.xml]: Done in bdbdaf4fc14742361f4eb0a858ed1fe79834c692

**Review by ShobhitAd — APPROVED** [review]

**Pull request merged**